robocop/run.py

```diff
@@ -32,6 +33,26 @@ def __init__(self, from_cli=False):
         self.load_reports()
         self.configure_checkers_or_reports()
 
+    def export_results(self, *issues):
+        """ Gather results from issues and reports, merge them in a list and return the data"""
+        payload = []
+        for report in self.reports:
+            output = report.get_report()
+            if output is not None:
+                payload.append(output)
```

Comment:
Do we need report output in payload? Because it will be inconsisted, ie
```
# issues alone 
[{'source': '/home/andrea/Documents/black-manta/test.robot', 'line': 0, 'col': 0, 'severity': 'W', 'rule_id': '0203', 'desc': 'Missing documentation in suite', 'msg_name': 'missing-doc-suite'}, {'source': '/home/andrea/Documents/black-manta/test.robot', 'line': 1, 'col': 0, 'severity': 'W', 'rule_id': '0509', 'desc': 'Section is empty', 'msg_name': 'empty-section'}, {'source': '/home/andrea/Documents/black-manta/test.robot', 'line': 1, 'col': 0, 'severity': 'W', 'rule_id': '1002', 'desc': 'Missing trailing blank line at the end of file', 'msg_name': 'missing-trailing-blank-line'}]
# and then reports
['Issues by ids:
W1007 (uneven-indent)                : 5
E0904 (nested-for-loop)              : 4
W0302 (not-capitalized-keyword-name) : 4

Found 12 issue(s): 9 WARNING(s), 4 ERROR(s).']
```
After receiving such list you would need to recognize and parse what's inside. It would be better to have only issues in output and then additionaly you can call report and get their output. Optionally reports could report json-like output to ie:
```
data = report.get_json_report()
print(data)
['W1007 (uneven-indent)': 5, 'E0904 (nested-for-loop)': 4]
# or different report
['total': 12, 'error': 4, 'warning': 8]
```